webant/agherant.py

```diff
@@ -36,7 +38,13 @@ def autoreplace(description_url):
             return url_for('description', _external=True)
         return description_url
     descriptions = map(autoreplace, descriptions)
-    clients = map(Client, descriptions)
+    clients = []
+    for url in descriptions:
+        try:
+            clients.append(Client(url))
+        except Exception as exc:
+            log.info("Error while retrieving description for '%s'" % url)
```

Comment:
reading from [python logger doc](https://docs.python.org/2/library/logging.html#logging.Logger.exception) and [stackoverflow](http://stackoverflow.com/questions/5191830/python-exception-logging)
I think you should write like this:
```python
log.exception("Error while retrieving description for '{}'".format(url))
```